packages/x-components/src/x-modules/search/store/__tests__/actions.spec.ts

```diff
@@ -57,14 +57,30 @@ describe('testing search module actions', () => {
   });
 
   describe('fetchAndSaveSearchResponse', () => {
-    it('should include the origin in the request', async () => {
+    it('should include the origin, start and rows properties in the request', async () => {
       resetSearchStateWith(store, { query: 'lego', origin: 'search_box:external' });
+      const { page, ...restRequest } = store.getters.request!;
       await store.dispatch('fetchAndSaveSearchResponse', store.getters.request);
 
       expect(adapter.search).toHaveBeenCalledTimes(1);
       expect(adapter.search).toHaveBeenCalledWith({
-        ...store.getters.request,
-        origin: 'search_box:external'
+        ...restRequest,
+        origin: 'search_box:external',
+        start: 0,
+        rows: 24
+      });
+    });
+
+    it('should calculate correctly the start and rows properties', async () => {
```

Comment:
I'll add another test for the case of start bigger than 0.

Reply:
As we are already testing `start: 0` in the previous test, I changed the `start` value here to something > 0.